### ISSUE TITLE
Q-Network

### ISSUE THREAD

**Issue by denizs** [denizs/torchUp#3]
The Q-Network is a neural network derived from `torch.nn.Module`, implementing all the features necessary for state action value estimation and back propagation by Gradient Descent.

**Comment by denizs** [comment]
Implemented in #5
